shell/common/gin_helper/event_emitter.h

```diff
@@ -108,8 +110,6 @@ class EventEmitter : public gin_helper::Wrappable<T> {
     // It's possible that |this| will be deleted by EmitEvent, so save anything
     // we need from |this| before calling EmitEvent.
     auto* isolate = this->isolate();
-    v8::Locker locker(isolate);
-    v8::HandleScope handle_scope(isolate);
```

Comment:
I'm not sure it's correct to remove this scope. Check all callers of `EmitWithEvent`?

Reply:
This method takes `v8::Local` as argument, so all callers are required to be in a handle scope and I think it is not needed here.

Reply:
Oh good point.